<launch>
  <node pkg="rr_platform" type="ultrasonic_array" name="ultrasonic_array_to_pcl" output="screen" respawn="false">
    <param name="number_of_sensors" type="int" value="6" />
    <param name="baud_rate" type="int" value="9600" />
    <param name="serial_port_name" type="string" value="/dev/ttyUSB0" />
    <param name="sensor_base_link" type="string" value= "/ultrasonic_array_base" />
    <param name="sensor_link_prefix" type="string" value= "ultrasonic_" />
    <param name="update_rate" type="double" value= "10.0" />
    <param name="draw_method" type="int" value="0" />
    <param name="number_of_points" type="int" value="5" />
    <param name="length" type="double" value="0.1" />
    <param name="radius" type="double" value="0.1" />
    <param name="distance_clip" type="double" value= "4.0" />
  </node>
</launch>
